frontend/Views/Widgets/ViewNemesisStats.svelte

```diff
@@ -0,0 +1,312 @@
+<script>
+    import { onMount } from "svelte";
+    import Chart from "chart.js/auto";
+
+    export let viewId;
+    export let dashboardObject;
+    let chartCanvas;
+    let chart;
+    let barChartCanvas;
+    let barChart;
+    let versions = [];
+    let releases = [];
+    let selectedFilter = "";
+    let totalDuration = 0;
+    let uniqueNemesisCount = 0;
+    let allNemesisData = [];
+    let errorMsg = "";
+    let loading = false;
+    let filteredData = [];
+
+    async function fetchData() {
+        try {
+            const response = await fetch(`/api/v1/views/widgets/nemesis_data?view_id=${dashboardObject.id}`);
+            if (!response.ok) {
+                throw new Error(`HTTP error! status: ${response.status}`);
+            }
+            const data = await response.json();
+            allNemesisData = data.response.nemesis_data || [];
+            return allNemesisData;
+        } catch (error) {
+            console.error("Failed to fetch nemesis data:", error);
+            errorMsg = "Failed to load Nemesis data.";
```

Comment:
If using a large block to show feedback to user, you can expand on the error cause - can use an object here.

Reply:
i added more info and traceId